specification/azure-kusto/resource-manager/readme.md

```diff
@@ -28,14 +28,23 @@ These are the global settings for the Kusto API.
 title: KustoManagementClient
 description: 'The Azure Kusto management API provides a RESTful set of web services that interact with Azure Kusto services to manage your clusters and databases. The API enables you to create, update, and delete clusters and databases.'
 openapi-type: arm
-tag: package-2019-09-07
+tag: package-preview-2020-01
```

Comment:
your default tag will now only have the languageExtension APIs and none of your other APIs.  Is that intended?  I believe there are other problems in the README but I'll defer to the SDK team on that

Reply:
You are right. I change the default version back to package-2019-09-07.

This (2020-01-01-preview) API version is preview and declare just preview APIs.